File: test_cases/python/claude/08.py
def run_tests(solution_module):
    f = solution_module.getPermutation
    # n=3, k=6 — last permutation
    assert f(3, 6) == '321'
    # n=2, first permutation
    assert f(2, 1) == '12'
    # n=2, last permutation
    assert f(2, 2) == '21'
    # n=4, k=1 — lexicographically first
    assert f(4, 1) == '1234'
    # n=4, k=24 — lexicographically last
    assert f(4, 24) == '4321'

File: test_cases/c/claude/08.c
#include <stdio.h>
#include <string.h>
#include <assert.h>

char* getPermutation(int n, int k);

int main(void) {
    assert(strcmp(getPermutation(3, 3), "213") == 0);
    assert(strcmp(getPermutation(4, 9), "2314") == 0);
    assert(strcmp(getPermutation(3, 1), "123") == 0);
    assert(strcmp(getPermutation(1, 1), "1") == 0);
    assert(strcmp(getPermutation(9, 362880), "987654321") == 0);
    return 0;
}